Source/WTF/wtf/spi/cocoa/MachVMSPI.h

```diff
@@ -39,6 +39,8 @@ kern_return_t mach_vm_allocate(vm_map_t target, mach_vm_address_t*, mach_vm_size
 kern_return_t mach_vm_deallocate(vm_map_t target, mach_vm_address_t, mach_vm_size_t);
 kern_return_t mach_vm_map(vm_map_t targetTask, mach_vm_address_t*, mach_vm_size_t, mach_vm_offset_t mask, int flags,
     mem_entry_name_port_t, memory_object_offset_t, boolean_t copy, vm_prot_t currentProtection, vm_prot_t maximumProtection, vm_inherit_t);
+kern_return_t mach_vm_remap(vm_map_t targetTask, mach_vm_address_t*, mach_vm_size_t, mach_vm_offset_t mask, int flags,
```

Comment:
Can we remove the duplicate SPI usage from ExecutableAllocator so it's obvious that we're not adopting any new SPI?

Reply:
Done in most recent push.